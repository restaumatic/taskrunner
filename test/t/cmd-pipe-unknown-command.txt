# Tests that we report a sensible error message when cmd pipe receives unknown command
echo '["random","command"]' >&${_taskrunner_request_pipe}
read -u ${_taskrunner_response_pipe} reply
eval "$reply"
