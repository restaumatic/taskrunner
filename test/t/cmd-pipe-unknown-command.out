-- output:
[toplevel] error | Unknown command in command pipe: ["random","command"]
-- exit code: 1
